tests/Behat/FieldsContext.php

```diff
@@ -100,4 +101,48 @@ public function selectNthOption(string $select, string $position, string $option
     $fields[$position]->selectOption($select);
   }
 
+  /**
+   * Step to fill in multi value fields with columns.
+   *
+   * @Given I fill in :column with :value in the :row :field field element
+   */
+  public function fillInMultivalueField($column, $value, $row, $field) {
```

Comment:
If we use in oe_paragraph these steps - it would be useful to adjust also EWCMS for avoiding duplications: https://github.com/ec-europa/ewcms/blob/develop/behat.yml.dist#L40